Jenkinsfile

```diff
@@ -1,19 +1,18 @@
 #!groovy
 
-// in case of change update method isMainBuild
-def jdks = ["jdk8","jdk9","jdk10","jdk11"]
+def mainJdk = "jdk8"
+def jdks = [mainJdk, "jdk9", "jdk10", "jdk11"]
```

Comment:
jdk9 and jdk10 are not removed?

Reply:
not yet in jetty-9.4.x (maybe could make sense? I don't really have strong opinion)

Reply:
Removing jdk9 and jdk10 will be a different PR/commit (it's a different issue: #2458)